src/components/Pricing/data.ts

```diff
@@ -0,0 +1,499 @@
+export interface FeatureCluster {
+    topic: string
+    features?: string[]
+}
+
+export interface FeatureInfo {
+    label: string
+    description?: string
+}
+
+interface FeatureDictionary {
+    topic: string
+    features: FeatureSubsetDictionary[]
+}
+
+interface FeatureSubsetDictionary {
+    label: string
+    business: boolean | string
+    enterprise: boolean | string
+    disclaimer?: string
+}
+
+/** Business spotlight feature set */
+export const BIZ_FEATURES_OVERVIEW: FeatureCluster[] = [
+    {
+        topic: 'Code intelligence platform',
+        features: [
+            'codeSearch',
+            'codeNavigation',
+            'batchChanges',
+            'codeInsights',
+            'notebooks',
+            'codeMonitoring',
+            'comprehensiveApi',
+        ],
+    },
+    {
+        topic: 'Code host integrations',
+        features: ['cloudHosts'],
+    },
+    {
+        topic: 'Security and admin',
+        features: ['ssoSaml', 'securityRoles', 'repoPerms', 'analytics'],
+    },
+    {
+        topic: 'Scale and performance',
+        features: ['businessStorage', 'businessExecutors'],
+    },
+    { topic: '24/5 support' },
+    { topic: 'Secure and single-tenant Cloud deployment' },
```

Comment:
```suggestion
    { topic: 'Secure, single-tenant Cloud deployment' },
```


Or just remove "secure" - isn't it implied? (I don't feel strongly about keeping vs. removing "secure". But the "and" sounds weird to me.)

Reply:
@st0nebraker - let's go with "Single-tenant Cloud deployment" - thank you!